src/i18n/fr.json

```
@@ -131,8 +131,12 @@
   },
   "dashboard": {
     "welcome": "Bienvenue",
-    "create": "Ajouter",
-    "my_items": "Mes évenements, endroits et organisateurs",
+    "create": {
+      "events": "Ajouter événement",
+      "places": "Ajouter endroit",
+      "organizers": "Ajouter organisateurs"
```

Since the other ones are also singular

```suggestion
      "organizers": "Ajouter organisateur"
```